getting-started/setup-building.rst

```diff
@@ -692,9 +692,17 @@ on Linux, macOS and iOS.
 
    For example, with **Homebrew**, install the dependencies::
 
-      $ brew install pkg-config openssl@3.0 xz gdbm tcl-tk
+      $ brew install pkg-config openssl@3.0 xz gdbm tcl-tk mpdecimal
```

Comment:
Similar changes should be made to the MacPorts recipe further down.

Reply:
Yes; addressed with 1a9eb7d0e723f75a8639cd5bcd08edb723f86fe3.